lib/rspec/shell/expectations/call_log.rb

```diff
@@ -26,8 +26,6 @@ def contains_argument_series?(*contains_arguments)
         end
 
         def get_argument_count(*expected_argument_series, sub_command_series: [], position: false)
-          expected_argument_series ||= []
-
```

Comment:
Not sure why this was here. expected_argument_series is not an optional parameter and I don't know how/why nil would be passed in